icons/hyundai.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Hyundai</title><path d="M11.999 18.145c6.627 0 12.001-2.751 12.001-6.144 0-3.395-5.374-6.146-12.001-6.146C5.374 5.855 0 8.606 0 12.001c0 3.393 5.374 6.144 11.999 6.144m2.26-3.015c-.2.464-.545 1.454-1.336 1.85-.24.119-.537.174-.8.185H12c-2.214 0-4.276-.31-6.002-.834l-.066-.025c-.16-.053-.24-.127-.24-.218 0-.079.043-.14.099-.195l.109-.097c.4-.332 1.598-1.2 3.858-2.067.793-.301 1.786-.679 2.825-.9.608-.126 2.868-.473 1.675 2.301m6.062-6.194c.043-.074.1-.137.203-.142.056-.006.132.007.248.08 1.409.867 2.245 1.952 2.245 3.125 0 2.118-2.724 3.94-6.62 4.735-.248.05-.416.048-.471-.015-.04-.038-.05-.106 0-.19a.815.815 0 01.104-.145c2.12-2.5 3.736-6.189 4.195-7.253.035-.074.068-.147.096-.195M9.777 8.857c.2-.463.545-1.454 1.335-1.846.24-.12.537-.178.8-.185.061-.002.104 0 .12 0 2.217 0 4.276.306 6.004.833.013.006.053.02.066.025.16.054.24.127.24.218 0 .079-.042.137-.098.193a1.89 1.89 0 01-.11.096c-.397.335-1.598 1.201-3.858 2.068-.795.304-1.786.679-2.822.899-.61.13-2.87.474-1.677-2.3M7.6 7.264c.25-.048.415-.048.476.015.035.04.045.106-.002.19a.89.89 0 01-.104.142c-2.12 2.503-3.737 6.189-4.198 7.256a2.313 2.313 0 01-.096.195c-.04.073-.099.136-.2.142-.056.005-.135-.011-.251-.081C1.817 14.256.98 13.172.98 11.999c0-2.118 2.724-3.94 6.62-4.735Z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Hyundai</title><path d="M20.734 7.687c1.615.906 3.443 2.499 3.252 4.622-.22 1.968-2.01 3.342-3.61 4.17-4.668 2.287-11.225 2.358-16.046.359C2.7 16.143.873 14.949.187 13.153c-.525-1.475.115-3.014 1.189-4.037C3.62 7.023 6.485 6.305 9.432 5.891c3.198-.399 6.565-.047 9.429.96.64.242 1.271.508 1.873.835v.001zM18.1 7.625c-1.912-.547-3.922-.875-6.055-.858-.99.015-1.706.96-2.11 1.764-.304.64-.654 1.405-.396 2.14a.974.974 0 0 0 .602.467c1.043.234 1.988-.188 2.924-.421 1.843-.617 3.649-1.32 5.225-2.584.068-.086.19-.172.129-.305-.068-.118-.205-.14-.32-.203zM8.16 7.25c-.053-.1-.19-.086-.297-.086-1.02.164-1.988.453-2.917.813-1.569.679-3.45 1.678-3.915 3.552-.289 1.436.716 2.646 1.828 3.396.22.1.457.414.701.18 1.25-2.694 2.566-5.403 4.57-7.738l.03-.117zm13.29 2.062c-.275-.188-.519-.461-.861-.492a.39.39 0 0 0-.168.116c-.37.843-.765 1.674-1.188 2.491-.921 1.82-1.972 3.615-3.359 5.215l-.022.11c.015.094.114.078.175.086.975-.093 1.874-.406 2.765-.71 1.637-.657 3.693-1.687 4.165-3.623.29-1.312-.563-2.397-1.508-3.194zm-6.938 4.137c-.122-.328-.465-.57-.808-.617-.83-.109-1.615.125-2.353.343-1.98.68-3.983 1.328-5.667 2.726-.075.07-.128.17-.09.28a.361.361 0 0 0 .212.157 21.01 21.01 0 0 0 5.865.921c.907.094 1.66-.53 2.117-1.256.411-.781.906-1.585.724-2.554z"/></svg>
```

Comment:
SVG available in the footer of the site gives [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/27a77111-90c1-4030-9700-22623ddd4976). Would you mind taking another look?

Reply:
I took the logo from header(hyundai-logo-wide-blue.svg) and on comparing it is different from the footer(hyundai-logo-stacked-white.svg).

Which one should we take?

Reply:
The logo in the header seems to be part of an icon font, which isn't easy to decompile (without a browser plugin!). I would say the footer is a more _accessible_ source for people wanting to review in future.